index.js

```diff
@@ -75,26 +61,26 @@ const renderSubMenu = (options, param2, index) => {
 
     subMenuElement.style.top = `calc(${index + 1} * ${elementHeight} + ${index + 1} * ${gapHeight})`
 
-    options.forEach(({ id, innerText, onclick }) => createElement("button", {
-        id,
+    options.forEach(({ innerText, onclick }) => createElement("button", {
         className: "sub-options",
         innerText,
         onclick
     }, subMenuElement))
 }
 
-
-// TODO: make 1 function from the 2
-const renderDigitalProgrammeList = () => {
+const renderNewScreen = () => {
     removeSubMenuElement()
-    handleOptionsElementOnClick("digitalProgrammeList")
-    console.log("Digital Programme List")
-}
 
-const renderDigitalEpg = () => {
-    removeSubMenuElement()
-    handleOptionsElementOnClick("digitalEpg")
-    console.log("Digital EPG")
+    mainElement.remove()
+
+    const screenWrapperElement = createElement("div", {
+        id: ELEMENT_IDS.screenWrapper
+    }, appElement)
+
+    const screenText = createElement("p", {
+        id: ELEMENT_IDS.screenText,
+        innerText: "???"
```

Comment:
Instead `???` you could have name of the screen you were going to present, right?

Reply:
That is what is said in the comment "I want to display the Inner Texts on the new screen, don't know how to achieve it."

Reply:
Solved